HelpSource/Classes/CombC.schelp

```diff
@@ -64,20 +64,25 @@ code::
 { CombN.ar(Decay.ar(Dust.ar(1,0.5), 0.2, WhiteNoise.ar), 0.2, 0.2, 3) }.play;
 
 // large delay lines
-(
-s.options.memSize= 8192*2;
+s.options.memSize = 8192 * 200; 
```

Comment:
sigh. we've been through this before.  have you already forgotten?  this is the third time you're suggesting changes to that example https://github.com/supercollider/supercollider/pull/2157#issuecomment-223759221 https://github.com/supercollider/supercollider/pull/2754#issuecomment-283316402
please be satisfied with what is there.  it is good. what you're suggesting here is not.  and if you can't explain why it's so bad then please go and do some supercollider studying and come back when you understand.